src/proposals/proposals.controller.ts

```diff
@@ -390,49 +420,51 @@ export class ProposalsController {
       "Database filters to apply when retrieving count for proposals",
     required: false,
     type: ProposalCountFilters,
-    example: `{ fields: ${proposalsFullQueryExampleFields}}`,
+    example: `{ fields: ${proposalsFullQueryExampleFields}, filter: ${filterExample}}`,
```

Comment:
Below is the value of the filterExample. Is it correct example for the filter field?
```
export const filterExample =
  '{ "where": { "field": "value" }, "include": [ { "relation": "target" } ], "fields": ["field1", "field2"], "limits": {"limit": 1, "skip": 1, "order": "asc"}}';
```